### ISSUE TITLE
Add static libs for arm64

### ISSUE THREAD

**Issue by tom-wp** [rticommunity/rticonnextdds-connector-go#48]
### Proposal:

Add Linux and Darwin `arm64` static libraries.

### Current behaviour:

The connector currently has static libraries for the following platforms ([rticonnextdds-connector-go/static_lib/](https://github.com/rticommunity/rticonnextdds-connector-go/tree/master/static_lib)):

* x64Darwin16clang8.0
* x64Linux2.6gcc4.4.5

### Desired behavior:

Add the following arm64 static library builds:
* arm64Darwin20clang12.0
* armv8Linux4gcc7.3.0

### Use case:

Static linking of projects using the connector on arm64 architectures.

**Comment by Arthurhussey** [comment]
+1

**Comment by tom-wp** [comment]
Thank you for clarifying.  Would it make sense to remove the static libraries from the next release to make it clear to developers that this is no longer supported?  Or at least add a readme to the static directory.

**Comment by kyoungho** [comment]
I agree. I think it makes sense to remove the static libraries from the next release. Thanks for your feedback.
